test/Atmos/EDMF/helper_funcs/subdomain_statistics.jl

```diff
@@ -0,0 +1,34 @@
+#### Helper kernels
+
+compute_subdomain_statistics(m::AtmosModel, state, aux, t) =
+    compute_subdomain_statistics(
+        m,
+        state,
+        aux,
+        t,
+        m.turbconv.micro_phys.statistical_model,
+    )
+
+"""
+    compute_subdomain_statistics(
+        m::AtmosModel{FT},
+        state::Vars,
+        aux::Vars,
+        t::Real,
+        statistical_model::SubdomainMean,
+    ) where {FT}
+
+Returns a cloud fraction and cloudy and dry thermodynamic
+states in the subdomain.
+"""
+function compute_subdomain_statistics(
+    m::AtmosModel{FT},
+    state::Vars,
+    aux::Vars,
+    t::Real,
+    statistical_model::SubdomainMean,
+) where {FT}
+    ts_en = recover_thermo_state_en(m, state, aux)
+    cloud_frac = has_condensate(ts_en) ? FT(1) : FT(0)
+    return (ts_en, ts_en, cloud_frac)
```

Comment:
Why do you have to return ts_en twice here?

Reply:
We're kind of looking forward. The `compute_subdomain_statistics` for `LogNormalQuad` will also return two thermo states, but they'll be different. We wanted to have an interface that should work for any statistical model.

Reply:
Anna , these two will be ts_cloudy and ts_dry states that differ, once the quadrature will be implemented.
Charlie, perhaps we can name them accordingly already now?